package-res/resources/web/prompting/api/OperationAPI.js

```diff
@@ -26,9 +26,9 @@
  * @property {Object} _msgs Contains possible constants of messages
  */
 define(['common-ui/prompting/PromptPanel', 'common-ui/prompting/parameters/ParameterXmlParser'], function(PromptPanel, ParameterXmlParser) {
-  return function(api) {
+  return function(api, id) {
     this._parameterParser = new ParameterXmlParser();
-    this._promptPanel = null;
+    this._promptPanel = new PromptPanel(id);
```

Comment:
if we are building a parameter panel without paramDefn will this https://github.com/pentaho/pentaho-platform-plugin-common-ui/pull/548/files#diff-fe97c9bc6920bca0d75fa6ad17207ee6R348 always fail? This check should probably not be here or this will never set the value of the auto submit